### ISSUE TITLE
resized picture are not sharp like it should

### ISSUE THREAD

**Issue by MegaV0lt** [linuxmint/pix#48]
```
 * pix 1.6.2, Copyright © 2001-2010 Free Software Foundation, Inc.
 * Distribution - Linux Mint 18.3 Sylvia
```

**Issue**
Resizing of picturers is blured.

**Steps to reproduce**
Take a picture (with text to see the result better). Resize it to 50% and keep aspect ratio. Save it as png. Redo the same with IView (Windows). When compare the two pictures. Result is that the resized picture from IView is noticable sharper. 

**Expected behaviour**
Pix should produde resized pictures of the same quality than the windows standard

**Other information**
Sample output of teh two resized pictures. Taken from an screen at ebay. Pix left - IView right
![auswahl_20180215_14 35 35](https://user-images.githubusercontent.com/2804301/36259458-895d3f16-125e-11e8-89cc-1ed2e0d70d90.png)

Same screenshot with 300% magnification:
![auswahl_20180215_14 37 06](https://user-images.githubusercontent.com/2804301/36259512-b2d8df76-125e-11e8-85ad-f4a30bcbcccd.png)

**Comment by icarter09** [comment]
@MegaV0lt are you still experiencing this issue?

**Comment by MegaV0lt** [comment]
Yes! Just made a new test. Compare the result of pix vs xnview! Pix makes pictures blurry when resizing Example was ~800 pixel and was resized almost 50 % smaller. See Result!

In the meantime i do not use pix anymore.for Editing
![Auswahl_20191217_08:03:47](https://user-images.githubusercontent.com/2804301/70972603-0e02a080-20a4-11ea-98b5-f483088b5298.jpg)

**Comment by icarter09** [comment]
@MegaV0lt thanks for the update.
